images/krte/Dockerfile

```diff
@@ -40,7 +40,7 @@ ENV IMAGE=${IMAGE_ARG} \
     CONTAINER=docker
 
 # copy in image utility scripts
-COPY ["wrapper.sh", "create_bazel_cache_rcs.sh", "install-bazel.sh", \
+COPY ["wrapper.sh", "create_bazel_cache_rcs.sh", "install-bazel.sh", "setup.sh", \
```

Comment:
can we please check https://github.com/kubernetes/test-infra/issues/18706 or fork this image?

Reply:
didn't want to lock into a specific  "kubetest2" name since this will become the entrypoint for all the kubetest2 jobs and could add more "setup" functionality later. 
but agree that `setup.sh` might be a bit too generic.